device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/qos.json.j2

```diff
@@ -1 +1,86 @@
+{% if ('type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'LeafRouter') or ('subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR') %}
+{%- macro generate_dscp_to_tc_map() %}
+    "DSCP_TO_TC_MAP": {
```

Comment:
Some of the mapping in these 2 tables "DSCP_TO_TC_MAP" and "TC_TO_PRIORITY_GROUP_MAP" are different if it used as a leafrouter vs dualToR.  you will need to separate out the check

if leaf:
define macro
elif dualtor
define macro